mod/beacon/blockchain/service.go

```diff
@@ -80,6 +80,12 @@ type Service[
 	// forceStartupSyncOnce is used to force a sync of the startup head.
 	forceStartupSyncOnce *sync.Once
 
+	// blobFinalized is used to verify blob sidecar availability upon
+	// block finalization
+	blobFinalized chan struct{}
```

Comment:
@itsdevbear we may want to harden this and pass some kind of blob data to enhance verification. Not sure what. You got suggestions?

Reply:
I think even making the data here be the slot number for which the blob belongs to is sufficient.